lib/assets/javascripts/cable.js.coffee

```diff
@@ -5,4 +5,8 @@
   PING_IDENTIFIER: "_ping"
 
   createConsumer: (url) ->
+    token = $("meta[name=csrf-token]").attr("content")
+    if token
+      separator = if url.indexOf("?")>=0 then "&" else "?"
```

Comment:
Need some spaces around the `>=`. And it might be nice to wrap up all this URL wrangling in a helper function.